multires-lyrics-search/app.py

```diff
@@ -41,14 +44,18 @@ def input_fn():
                     d.tags['SName'] = row[1]
                     d.tags['SLink'] = row[2]
                     d.text = row[3]
-                yield d
+                docs.append(d)
+
+    return docs
```

Comment:
any reason we want to use list instead of generator?

Reply:
I think because its neat to get the total size upfront for the logging, but I guess its not really necessary.

Reply:
oh ok